.travis.yml

```diff
@@ -10,6 +10,7 @@ before_install:
   - sudo mv docker-compose /usr/local/bin
 
 script:
+  - sed -i "s/NODE_ENV=/NODE_ENV=$(echo $NODE_ENV)/" .env
```

Comment:
Replaces the line in `NODE_ENV=` in `.env` with whatever the TravisCI environment variable is set to so docker builds the for the right environment.

The `s/` command is substitution... so `s/<what we are looking to replace>/<what we want to replace it with>/`